### ISSUE TITLE
Feature request: Browser key shortcut to change table/view

### ISSUE THREAD

**Issue by tonypdmtr** [sqlitebrowser/sqlitebrowser#536]
When in the browser tab, CTRL-UpArrow or CTRL-DnArrow currently behave (AFAIK) like simple Up/DnArrow except for moving the marker.

I propose these keyboard shortcuts be used instead for selecting previous next table/view (in the order these appear in the drop-down list above but order is not really of much importance).

This makes it very fast (for those of us who prefer the keyboard over the mouse) to change table/view.

Thank you.

**Comment by justinclift** [comment]
Sounds useful. :smile:

**Comment by MKleusberg** [comment]
I've just made a commit that adds two new shortcuts for this. For now I have decided to set them to ```Ctrl + PageUp``` and ```Ctrl + PageDown``` because in my opinion Ctrl+Arrow keys feels like it's about selecting table cells (even though it doesn't at the moment), not like changing the table altogether. But that's just my opinion. If a majority feels like it should be the arrow keys I can always change the shortcuts :smiley:

Anyway, are you still interested in this, @tonypdmtr, and want to check out tomorrow's nightly build to see if it's working for you?
